### PR TITLE
#2404 Process about image mgt changes

### PR DESCRIPTION
**Pull request by lfasani**

### CODE REVIEW

**Comment by lfasani** [comment]
Do not merge this PR because there is a bug in path conversion for image created from base64

**Review by pdulth — CHANGES_REQUESTED** [review]

File: tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/i/Rule_I_47_IncorrectImagePath.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    CapellaModel model = getTestModel(getRequiredTestModel());
+    IScope scope = new ScopeModelWrapper(model);
+    Part part = (Part) IdManager.getInstance().getEObject(INVALID_LC2_PART, scope);
+    part.eResource().eAdapters().removeIf(DataNotifier.class::isInstance);
+    ExecutionManager manager = ExecutionManagerRegistry.getInstance().getExecutionManager(model.getEditingDomain());
+    manager.execute(new AbstractReadWriteCommand() {
+      @Override
+      public void run() {
+        part.setDescription(
+            "&lt;p>&lt;img src=&quot;file:/C:/Users/T0108576/Documents/Docs/Docs/11_Images/circle_ws2.png&quot; />&lt;/p>&#xA;");
```
[the remark this answers is not in the document]
Comment (pdulth) [reply tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/i/Rule_I_47_IncorrectImagePath.java:79]: I think you can override the diagnotician for this test by overriding the create diagnotician and use a CapellaDiagnostician
https://github.com/eclipse/capella/blob/master/tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/ValidationRuleTestCase.java#L151

something like 
new CapellaDiagnostician((TED)manager.getEditingDomain().getAdapterFactory(), new NullProgressMonitor());

[the remark this answers is not in the document]
Comment (lfasani) [reply tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/i/Rule_I_47_IncorrectImagePath.java]: Done

**Review by pdulth — APPROVED** [review]

**Comment by pdulth** [comment]
I updated TOC

**Pull request merged**